### ISSUE TITLE
[FEATURE_REQUEST] iOS support?

### ISSUE THREAD

**Issue by mrpotatocode** [mateusz-bak/openreads-android#149]
Fantastic app -- I'm very much enjoying it. Thank you for your amazing work.

Unfortunately my other half has an iphone and is jealous. 
I understand this is a huge ask, but no one seems to have asked before, and I wanted to express interest! 

Maybe somewhere later down the road? 🤞

**Comment by mateusz-bak** [comment]
Hi @mrpotatocode the development work is done. Only thing left is to gather necessary funding for some macos dev time ~$30 and $200 for Apple Developer account.

I invite you and everyone interested to pitch in:

https://github.com/sponsors/mateusz-bak
https://www.buymeacoffee.com/mateuszbak

**Comment by gbertolottiKS** [comment]
Hi.
How the funding is going? I can do my part, if we're close enough…
Thanks for your work!

**Comment by mateusz-bak** [comment]
Hi @gbertolottiKS so far we are at $102!

**Comment by rpoovey** [comment]
Is it possible that you could post the .ipa package so we could just sideload with Altserver or something else?

**Comment by mateusz-bak** [comment]
@rpoovey  As far as I know signing .ipa packages requires signing by a paid dev account.

I'll be setting up the dev account soon but I need to see if distributing the app outside App Store won't get me banned.

**Comment by rpoovey** [comment]
@mateusz-bak I do not believe that to be true. You can compile the ipa without a paid account and then someone could then use [Altstore](https://altstore.io/) with that ipa and get it resigned with their AppleID to then sideload it. But its all moot anyway. The paid account to publish to the app store is only $99 a year and it seems you've hit over that as of Aug 22 comment. So the App Store method would be the easiest and most above board.

**Comment by mvolpato** [comment]
I am not sure you can publish on the Apple AppStore. Apparently GPL is not compatible with the AppStore, so you would need to re license it. When relicensing it you will need all contributors to wave their contribution to you.

**Comment by mateusz-bak** [comment]
@mvolpato I saw some discussions on this topic  but this is still not clear to me as I'm not a licencing expert. Would you have an advice how to proceed?

Anyway as I finished with the iOS compatibility I plan to pay for the account and create a Test Flight release in the following week or two.

**Comment by mvolpato** [comment]
I am not an expert either. The point is that IF you own the code, you can distribute the app as many times as you want with as many licenses as you want.

But, do you own the code now? Do you own code written by other contributors?

**Comment by mateusz-bak** [comment]
Definietely 99% of the code. I see two options:
[Add a clause for the App Store DIstribution](https://opensource.stackexchange.com/questions/8674/gpl-with-license-exception-for-ios)
Or re-licence to MIT

**Comment by mateusz-bak** [comment]
I invite everyone to help with iOS testing in TestFlight 📱 🍎 

Contact me through the [Matrix channel](https://matrix.to/#/#openreads:matrix.org)

**Comment by mateusz-bak** [comment]
Openreads released to App Store
